centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/testUtils.ts

```diff
@@ -210,7 +190,7 @@ const platformVersions = {
 };
 
 const formData = {
-  contactgroups: [1],
+  contactgroups: [],
```

Comment:
You can remove this line, remove the contactgroups decoder to enable that, along with the contactgroups type in the Notification type